chunky/src/java/se/llbit/chunky/ui/render/WaterTab.java

```diff
@@ -142,12 +142,12 @@ public void initialize(URL location, ResourceBundle resources) {
     waterPlaneEnabled.selectedProperty().addListener((observable, oldValue, newValue) ->
       scene.setWaterPlaneEnabled(newValue)
     );
+
     waterPlaneHeight.setName("Water height");
     waterPlaneHeight.setTooltip("The default ocean height is " + World.SEA_LEVEL + ".");
     waterPlaneHeight.clampBoth();
     waterPlaneHeight.onValueChange(value -> scene.setWaterPlaneHeight(value));
-    Slider slider = waterPlaneHeight.getSlider();
-    slider.setBlockIncrement(0.5);
```

Comment:
Missed this yesterday somehow. We could add a `setStep` method to our numeric slider.

Reply:
This code sadly does not provide stepping in the way I thought - only for slider movement via arrow keys. We could enable snapping but then values between steps would not be possible. 
I'd say this is not needed in this PR ;)